src/multipart_.rs

```diff
@@ -7,6 +7,10 @@ use std::path::Path;
 use mime_guess::{self, Mime};
 use url::percent_encoding;
 use uuid::Uuid;
+use http;
+
+/// An alias to http::HeaderMap so http is unneded as an explicit dependency
+pub type HeaderMap = http::HeaderMap<String>;
```

Comment:
I'd remove this alias, and use the default `http::HeaderMap` which uses `HeaderValue` instead of `String`.